client/keys/export.go

```diff
@@ -25,13 +29,31 @@ func ExportKeyCommand() *cobra.Command {
 				return err
 			}
 
-			armored, err := clientCtx.Keyring.ExportPrivKeyArmor(args[0], encryptPassword)
-			if err != nil {
-				return err
+			hex, _ := cmd.Flags().GetBool(flagHex)
+
+			if !hex {
+				armored, err := clientCtx.Keyring.ExportPrivKeyArmor(args[0], encryptPassword)
+				if err != nil {
+					return err
+				}
+
+				cmd.Println(armored)
+				return nil
+			} else {
```

Comment:
The else block is unnecessary here cause the `if` block ends with a return statement.

Reply:
Removed